.github/workflows/check-spelling.yml

```diff
@@ -16,7 +16,7 @@ jobs:
             content/**/*.md
             layouts/**/*.md
             data/**/*
-            !data/registry
+            !data/registry/**/*
```

Comment:
I'm not convinced that this is actually needed, b/c we're not seeing any errors being reported in this PR, only in the JS PR. But it shouldn't hurt.